usagov/vote-gov-drupal
# Bug/vote-233/bengali numbers

## Jira ticket
https://cm-jira.usa.gov/browse/VOTE-233)

## Description

Fix Bengali number translation bug
Fix registration date translation formats

## Deployment and testing

### Post-deploy
1. lando retune

### QA/Test
1. Edit the [Alaska page](https://vote-gov.lndo.site/bn/register/ak) to use the specific registration date fields
2. Verify the registration date day names and numbers translate
3. Verify the "last update date" translates the Bengali numbers

## Checklist for the Developer




## Checklist for the Peer Reviewers
